pages/participate/participate.md

```diff
@@ -4,8 +4,61 @@ title: Participate
 description: 
 menubar: participate
 permalink: participate/
+menubar_toc: true
 set_last_modified: true
 ---
 
-More information to come...
+# Call for Submissions
 
+We invite submissions for the second annual US-RSE Conference to be held in
+Albuquerque, New Mexico, October 15-17, 2024. The theme of this year’s conference
+is “Yesterday, Today, Tomorrow: A celebration of all that RSEs have done for
+computing in the past, in the present, and in the future.” Topics of interest
+include, but are not limited to:
+
+- History of the RSE movement
+- RSE impact
+- Efforts to expand the RSE movement
+- Research data management
+- Software engineering approaches supporting research
+- Diversity, Equity, and Inclusion for RSEs and in RSEng
+- Training and workforce development
+- Maturing and expanding the RSE profession
+
+Accepted abstracts will be placed on the website ahead of the conference.
+
+## Important Dates
+
+- **Submissions open**: Monday, February 12 for all but notebooks. Notebooks to open Monday, February 26.
+- **BoFs, Workshops, and Tutorials**: Due March 22. Notifications April 26.
+- **Papers and Notebooks**: Due May 20. Notifications June 27.
+- **Posters and Talks**: Due June 3. Notifications July 3.
+- **Camera-ready submissions (Notebooks, Papers)**: August 9
```

Comment:
Please add days of the week (e.g., "Friday, March 22"). The extra clarity never hurts!